daemons/dss-notify-v2/app.py

```diff
@@ -105,5 +102,15 @@ def launch_from_notification_queue(event, context):
         else:
             logger.warning(f"Recieved queue message with no matching subscription:{message}")
 
+def _notify_subscribers(replica: Replica, key: str):
+    metadata_document = build_bundle_metadata_document(replica, key)
+
+    def _func(subscription):
+        if should_notify(replica, subscription, metadata_document, key):
+            notify_or_queue(replica, subscription, metadata_document, key)
+
+    with ThreadPoolExecutor(max_workers=20) as e:
```

Comment:
Factor of the number of available cores?

Reply:
added todo comment